### PR TITLE
Added lucacrypto

### PR DESCRIPTION
**Pull request by lucadice**

### CODE REVIEW

**Comment by lucadice** [comment]
https://polkadot.subscan.io/extrinsic/17261978-2

File: registry/polkadot.json [hunk 1]
```diff
@@ -295,5 +295,13 @@
...
         "longDescription": "## About me\n\nPolkadot Ambassador and early Polkadot Content Creator.\n\n## Contact me\n\nTelegram: KryptosChain\n\n[Twitter: @KryptosChain](https://twitter.com/KryptosChain)",
         "isOrganization": false
+    },
+]
+        "address": "14g2mcpaEJoMCpnzStbYTTRKcZsWAYeKUzdE8ph6jYCrZuPS",
+        "name": "LucaCrypto",
+        "image": "https://raw.githubusercontent.com/lucadice/opengov-delegate-registry/master/images/Lucacrypto.png",
+        "shortDescription": "Vibing and voting 🕺🗳️ Polkadot Enthusiast 🚀🔵 long time viewer of AAG 👁️📺[Twitter: @NFTsaremyjam](https://twitter.com/NFTsaremyjam)",
+        "longDescription": "Vibing and voting 🕺🗳️ Polkadot Enthusiast 🚀🔵 long time viewer of AAG 👁️📺[Twitter: @NFTsaremyjam](https://twitter.com/NFTsaremyjam)",
+        "isOrganization": false
```
Comment (stepanLav) [inline registry/polkadot.json:305]:
```suggestion
    },
    {
        "address": "14g2mcpaEJoMCpnzStbYTTRKcZsWAYeKUzdE8ph6jYCrZuPS",
        "name": "LucaCrypto",
        "image": "https://raw.githubusercontent.com/lucadice/opengov-delegate-registry/master/images/Lucacrypto.png",
        "shortDescription": "Vibing and voting 🕺🗳️ Polkadot Enthusiast 🚀🔵 long time viewer of AAG 👁️📺[Twitter: @NFTsaremyjam](https://twitter.com/NFTsaremyjam)",
        "longDescription": "Vibing and voting 🕺🗳️ Polkadot Enthusiast 🚀🔵 long time viewer of AAG 👁️📺[Twitter: @NFTsaremyjam](https://twitter.com/NFTsaremyjam)",
        "isOrganization": false
```

File: images/HOxWY_k2_400x400.jpg
Comment (stepanLav) [inline images/HOxWY_k2_400x400.jpg:1]: you have submitted two the same pictures, please remove one of it

**Comment by stepanLav** [comment]
@lucadice 
Hello! Thank you for request!
This is how it will look like in the app:
<details>
  <summary>App screens</summary>

Screenshot 2023-09-25 at 09 04 59

Screenshot 2023-09-25 at 09 04 56 copy



</details>

**Review by stepanLav — APPROVED** [review]

**Pull request merged**